### PR TITLE
Add "-total" metric keys for schema-dependent metrics

### PR DESCRIPTION
**Pull request by Dakostu**
This PR adds three metric keys `catalog.num-partitions-total`, `catalog.num-events-total` and `ingest-total` that sum up all schema-based metrics by their respective schema-based metric counterparts.

### :memo: Reviewer Checklist

Review this pull request by ensuring the following items:

- [x] All user-facing changes have changelog entries
- [x] User-facing changes are reflected on [vast.io](https://github.com/tenzir/vast/tree/master/web)

### CODE REVIEW

File: libvast/src/system/catalog.cpp [hunk 1]
```diff
@@ -516,27 +516,41 @@ catalog(catalog_actor::stateful_pointer<catalog_state> self,
         num_partitions += 1;
         num_events += synopsis->events;
       }
+      auto total_num_partitions = count{0};
+      auto total_num_events = count{0};
       auto r = report{};
       r.data.reserve(num_partitions_and_events_per_schema_and_version.size());
       for (const auto& [schema_and_version, num_partitions_and_events] :
            num_partitions_and_events_per_schema_and_version) {
+        auto schema_num_partitions = num_partitions_and_events.first;
```
Comment (patszt) [inline libvast/src/system/catalog.cpp:525]: Could be changed to one liner:
auto [num_partitions, num_events] = num_partitions_and_events;

But tbh it's not a big deal. Pick the one you like more :)

File: libvast/src/system/importer.cpp [hunk 2]
```diff
@@ -153,9 +153,14 @@ void importer_state::send_report() {
...
   samples.push_back(performance_sample{"node_throughput"s, node_throughput});
-  for (const auto& [name, count] : schema_counters)
+  auto total_count = count{0};
+  for (const auto& [name, count] : schema_counters) {
+    total_count += count;
     samples.push_back(performance_sample{
       "ingest", measurement{elapsed, count}, {{"schema", name}}});
+  }
+  auto total_measurement = measurement{elapsed, total_count};
+  samples.push_back(performance_sample{"ingest-total"s, total_measurement});
```
Comment (patszt) [inline libvast/src/system/importer.cpp:163]: In the story it was suggested to name the key ingest.rate-total. I think we should stick with it

**Review by patszt — CHANGES_REQUESTED** [review]
Congrats on first PR. Please verify the naming of the ingest-total.
Other than that the changes look good

File: libvast/src/system/catalog.cpp (hunk 1 above)
Comment (Dakostu) [reply libvast/src/system/catalog.cpp:525]: Structured bindings are "more modern" and already used in the rest of the file - I applied your suggestion.

File: libvast/src/system/importer.cpp (hunk 2 above)
Comment (Dakostu) [reply libvast/src/system/importer.cpp:163]: The log will produce the following values for the `ingest-total` `performance_sample`: 
Screenshot_20221104_124521

So we would have `ingest-total.rate` amongst other things.

It looks like those three suffixes are added by the `accountant_actor`: 
Screenshot_20221104_124632

Do you think adding extra logic for this edge case (the `performance_sample` is total) would be worth it?

Comment (patszt) [reply libvast/src/system/importer.cpp:163]: Okay, then it is not worth it

**Review by patszt — APPROVED** [review]

**Pull request merged**